dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -1578,6 +1585,25 @@ public void cleanup(){
 		getWritableNullProperties().clear();
 	}
 
+	@Override
+	public ManifestInfo getManifestInfo() {
+		String type = null;
```

Comment:
you can reduce to

final String type = Host.class.isInstance(this.getClass()) || this.isHost()? PusheableAsset.SITE.getType():PusheableAsset.CONTENTLET.getType();

Reply:
done https://github.com/dotCMS/core/pull/20632/commits/c4232e449434945624566c867c3f524f4df32cb0